tempesta_fw/http_sess.c

```diff
@@ -59,15 +60,29 @@
  * @name		- name of sticky cookie;
  * @name_eq		- @name plus "=" to make some operations faster;
  * @sess_lifetime	- session lifetime in seconds;
+ * @max_misses		- maximum count of requsts with invalid cookie;
+ * @timeout		- maximum time to wait the request with valid cookie;
```

Comment:
Please, add the unit name for the member: jiffies or second. Now it's required to look though the code to identify it.

Reply:
Done.